docs/source/integrations/middleware.md

```diff
@@ -51,16 +162,20 @@ async function startApolloServer(typeDefs, resolvers) {
 }
 ```
 
-While it's easy to get started with `apollo-server` because you don't have to worry about a web framework, you can't tweak  its behavior as much as you can with web framework integrations. For example, you can't serve other endpoints from the same HTTP server. If you want to do something with your server that isn't supported by `apollo-server`, you can "eject" by switching over to `apollo-server-express`. (`apollo-server` is just a thin wrapper that combines `apollo-server-express` with `express`.)
+Although `apollo-server` helps you get started fast, you can't configure its behavior as much as you can other Apollo Server packages. For example, you can't serve other endpoints from the same HTTP server.
 
-## `apollo-server-express`
+If you want to do something with your server that isn't supported by `apollo-server`, you can [swap to `apollo-server-express`](#swapping-out-apollo-server).
 
-`apollo-server-express` is the GraphQL server for [Express](https://expressjs.com/), the most popular Node.js web framework. `apollo-server-express` lets you attach a GraphQL server to an Express server. `apollo-server` is built on top of `apollo-server-express`; if you start with `apollo-server` and later need to tweak how it serves over HTTP, you can easily "eject" from `apollo-server` by porting your server from `apollo-server` to `apollo-server-express`.
+### `apollo-server-express`
+
+`apollo-server-express` is the Apollo Server package for [Express](https://expressjs.com/), the most popular Node.js web framework. It enables you to attach a GraphQL server to an existing Express server.
+
+`apollo-server` uses `apollo-server-express` under the hood. If you start with `apollo-server` and later need to modify how it serves over HTTP, you can [swap `apollo-server` to `apollo-server-express`](#swapping-out-apollo-server).
 
 The following function is roughly equivalent to the `apollo-server` sample code above.
 
-```
-$ npm install apollo-server-express express graphql
+```bash
+npm install apollo-server-express express graphql
```

Comment:
I can't put the comment directly on it but yikes, the line `const { url } = await server.listen();` should NOT be in the apollo-server-express example